### DISCUSSION TITLE
how can i extract_wisdom from an audio file ?

### DISCUSSION THREAD

**Discussion by Haqbani** [danielmiessler/Fabric#502] [Q&A] [answered]
hello everyone! i have installed the need packeges for audio, however, i found if difficult  to write the code to extract_wisdom ?

any help ?

**Comment by black-backdoor** [comment, marked as answer]
Hi
I think you can use the ts helper app for this. Check out the "ts (Audio transcriptions)" section in the [README.md](https://github.com/danielmiessler/fabric?tab=readme-ov-file#ts-audio-transcriptions) file for more details.

**Reply by phillipadsmith** [reply]
@black-backdoor It seems that there might not be a "ts (Audio transcriptions)" section in the [README.md](https://github.com/danielmiessler/fabric?tab=readme-ov-file#ts-audio-transcriptions) anymore?

**Reply by black-backdoor** [reply]
Hi @phillipadsmith,  
Looking at the commit history, I believe I was referring to this section in the [README.md](https://github.com/danielmiessler/fabric/tree/82b99e9b13ea27f4ee0785ce691c55f9ccd085bf?tab=readme-ov-file#helper-apps).  
It’s been a while since I last used Fabric—does the `ts` helper app no longer exist?

**Reply by phillipadsmith** [reply]
Hi @black-backdoor Seems that it does not exist anymore, as far as I can tell...?
